CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ message(STATUS "Shaderc: build type is \"${CMAKE_BUILD_TYPE}\".")
 
 option(SHADERC_ENABLE_SPVC "Enable libshaderc_spvc" OFF)
 option(SHADERC_ENABLE_SPVC_PARSER "Enable spvc parser \(WIP)" OFF)
+option(DISABLE_EXCEPTIONS "disable spirv-cross exceptions")
```

Comment:
This looks like it should go into third_party/CMakeLists.txt just before we include the spirv-cross subdirectory?

Otherwise the name looks too generic, applying to all code everywhere.